api/v1/serviceinstance_types.go

```diff
@@ -82,6 +82,9 @@ type ServiceInstanceSpec struct {
 	// end-user. User-provided values for this field are not saved.
 	// +optional
 	UserInfo *v1.UserInfo `json:"userInfo,omitempty"`
+
+	// The subaccount id of the service instance
```

Comment:
The desired subaccount id for the service instance

Reply:
done